src/components/country-labels-layer/country-labels-layer-component.jsx

```diff
@@ -1,16 +1,23 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { loadModules } from 'esri-loader';
 import { handleLayerCreation } from 'utils/layer-manager-utils';
-import { COUNTRIES_LABELS_FEATURE_LAYER } from 'constants/layers-slugs';
+import { COUNTRIES_LABELS_FEATURE_LAYER, GRAPHIC_LAYER } from 'constants/layers-slugs';
+import { findLayerInMap } from 'utils/layer-manager-utils';
+import { simplePictureMarker } from 'utils/graphic-layer-utils';
+import mapPinIcon from 'icons/map_pin.svg'
 import { layersConfig } from 'constants/mol-layers-configs';
 
 const CountryLabelsLayerComponent = props => {
```

Comment:
Hey @weberjavi, do you think it makes sense to have .jsx file even if we don't render anything here? Maybe you could move it to the `.js` container file and remove this one?

Reply:
So as I see it we are following the same pattern as in other components even if we are not returning any `html`. We are handling layer fetching and events on the `container` and rendering stuff on the screen on the `component`. But yeah, maybe we can move everything to the `js` file, since the `jsx` extension might be confusing (I was also trying to avoid the file growing a lot, but maybe moving some of this logic of labeling and painting to some utils would make the `js` smaller and clearer).

Reply:
I see your point, feel free to choose the approach you want ;)